Cargo.toml

```diff
@@ -162,6 +164,12 @@ harness = false
 name = "send_events"
 harness = false
 
+[workspace.dependencies]
+anyhow = "1"
+once_cell = "1.18.0"
+regex = "1.10"
+rusqlite = { version = "0.31" }
```

Comment:
Specifying the version here lets workspace members say "I'd like the same version, please" with `workspace = true`, this prevents out-of-sync version numbers.

Reply:
so this states a preference but versions will be resolved across all crates of the workspace?

Reply:
It's more than just a preference (if I understood [the documentation](https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-dependencies-table) correctly, which I do assume) - since it says `regex = "1.10"` here, writing `regex = { workspace = true }` in a workspace member is equivalent to writing `regex = "1.10"`